contracts/evm/contracts/xcall/CallService.sol

```diff
@@ -635,8 +635,8 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
         string[] memory _sources
     ) external view override returns (uint256) {
         uint256 fee = protocolFee;
-        if (isReply(_net, _sources) && !_rollback) {
-            return fee;
+        if (isReply(_net, _sources)) {
```

Comment:
Should it not still be && !_rollback ?

Reply:
Ah, I guess it should be, I probably checked with java as there is not check for rollback, and changed I guess, so need to change in both right?

Reply:
Yes, It seems to have been missed in Java